units/src/amount/unsigned.rs

```diff
@@ -519,14 +519,14 @@ impl<'a> core::iter::Sum<&'a Amount> for Amount {
         I: Iterator<Item = &'a Amount>,
     {
         let sats: u64 = iter.map(|amt| amt.0).sum();
-        Amount::from_sat(sats)
+        Self::from_sat(sats)
     }
 }
 
 #[cfg(feature = "arbitrary")]
 impl<'a> Arbitrary<'a> for Amount {
     fn arbitrary(u: &mut Unstructured<'a>) -> arbitrary::Result<Self> {
         let a = u64::arbitrary(u)?;
-        Ok(Amount(a))
+        Ok(Self(a))
```

Comment:
I like this change but I believe blindly comparing two different things and saying they're OK because they are equal doesn't really work. So maybe don't make such claims in the commit message and simply say "for consistency".

Reply:
Oh I meant that one can have the two modules open side by side and scan down the code and the differences jump out more easily if everything that is non meaningful is exactly the same.

Reply:
Well, it's a way to understand how the modules are different but not a way to audit if one of them is correct based on assumption that another one is. :)

Reply:
Certainly, gratuitous text differences won't *help*.

It might be an interesting attempt to look for places where the signed/unsigned logic differ, and to use the explicit type name rather than `Self` in exactly those locations.

But I think this change is an improvement nonetheless.